### ISSUE TITLE
App crashes when calling NSDate Collection methods

### ISSUE THREAD

**Issue by dingua** [min/MNCalendarView#3]
The application launched from XCode 5 used to crash on every NSDate  collection method.
The log of crash is :"[__NSDate mn_beginningOfDay:]: unrecognized selector sent to instance"
Is it related to project integration on XCode 5?

**Comment by min** [comment]
I don't have this issue on XCode5, is this still an issue for your?

**Comment by dingua** [comment]
Yes, issue doesn't exist, i still have problem of integration of your framework in my project.. but it is caused by our project configuration. I used pods for framework integration .This works when making it with a new empty project but with my working project i still have the same problem.
Do you have an idea what means this crash? Because it compiles well, but once i call any method of your framework's classes it crashes.
Thanks.

**Comment by ghost** [comment]
@dingua you have to add MKCalendarView files to dedicated group of project's sources. Don't add it as a framework, just add files to project as i showed in screenshot.
![2013-10-04 0 29 14](https://f.cloud.github.com/assets/5605787/1265157/8c90a562-2c6a-11e3-9a92-121e78668972.png)

Your problem causes because MKCalendar extends NSDate object with some methods. So your project can not resolve this methods.
